app/views/questionnaire.py

```diff
@@ -462,35 +443,33 @@ def _get_schema_context(full_routing_path, metadata, collection_metadata, answer
                                 answer_ids_on_path=answer_ids_on_path)
 
 
-def get_page_title_for_location(schema, current_location, metadata, answer_store):
+def get_page_title_for_location(schema, current_location, context):
     block = schema.get_block(current_location.block_id)
     if block['type'] == 'Interstitial':
         group = schema.get_group(schema.get_group_by_block_id(block['id'])['id'])
         page_title = '{group_title} - {survey_title}'.format(group_title=group['title'], survey_title=schema.json['title'])
```

Comment:
I know this is slightly out of the context of this PR, but using the `group_title` in the title for an interstitial page seems odd. I think it's the only place we use a groups title where we should probably use the block title.
This also looks odd as the page title does not really have anything to do with the heading on the page

Reply:
I can remove it if there are no objections?